src/components/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const Breadcrumbs = ({ parent, parentText, rendered }) => {
+  return (
+    <p className="breadcrumb">
+    <Link className="breadcrumb__item" to="/">
+      Home
+    </Link>
+    /
+    <Link className="breadcrumb__item" to={ `/${parent}` } >
+      { parentText }
+    </Link>
+    /<span className="breadcrumb__item breadcrumb__item--selected">{rendered}</span>
+  </p>
+  )
+}
+
+export default Breadcrumbs;
```

Comment:
Github likes empty lines at the bottom of files. Could you config your editor to do this?

<img width="377" alt="Screen Shot 2021-01-25 at 1 38 36 PM" src="https://user-images.githubusercontent.com/13306885/105769407-a065ac80-5f12-11eb-99fa-30727bab8470.png">



```suggestion
export default Breadcrumbs;

```